ocaml/forkexecd/lib/forkhelpers.ml

```diff
@@ -179,12 +220,9 @@ type syslog_stdout =
   | Syslog_DefaultKey
   | Syslog_WithKey of string
 
-(** Safe function which forks a command, closing all fds except a whitelist and
-    having performed some fd operations in the child *)
-let safe_close_and_exec ?tracing ?env stdin stdout stderr
+let safe_close_and_exec_daemon ?tracing env stdin stdout stderr
     (fds : (string * Unix.file_descr) list) ?(syslog_stdout = NoSyslogging)
-    ?(redirect_stderr_to_stdout = false) (cmd : string) (args : string list) =
-  with_tracing ~tracing ~name:__FUNCTION__ @@ fun tracing ->
```

Comment:
Did you intend to remove the tracing from here?

Reply:
Yes, code is actually not removing it, just moved on the upper layer, see caller.